### PR TITLE
README: Add description and link to Code of Conduct.

### PR DESCRIPTION
**Pull request by microbit-pauline**
Code of Conduct added to README

### CODE REVIEW

**Comment by microbit-carlos** [comment]
@dpgeorge we are in the process of adding this CoC paragraph and link to the Foundation repositories.
Taking in consideration Micropython also has a CoC I assume this is okay with you as well?

File: README.md [hunk 1]
```diff
@@ -32,3 +32,8 @@ port.  To test it you can run:
 
     >>> display.show(Image.HAPPY)
     >>> audio.play(Sound.HAPPY)
+    
+Code of Conduct
+-------------------
+
+Trust, partnership, simplicity and passion are our core values we live and breathe in our daily work life and within our projects. Our open-source projects are no exception. We have an active community which spans the globe and we welcome and encourage participation and contributions to our projects by everyone. We work to foster a positive, open, inclusive and supportive environment and trust that our community respects the micro:bit code of conduct. Please see our [code of conduct](https://microbit.org/safeguarding/) which outlines our expectations for all those that participate in our community and details on how to report any concerns and what would happen should breaches occur.
```
Comment (microbit-carlos) [inline README.md:39]: @microbit-pauline the rest of this readme stays at 79 chars max per line, so if you could break down this paragraph with that line length that would be great. Thanks!

Comment (microbit-pauline) [reply README.md:39]: Will do

**Comment by dpgeorge** [comment]
> Taking in consideration Micropython also has a CoC I assume this is okay with you as well?

Yes, this is OK.

**Pull request merged**